src/granite/error.cr

```diff
@@ -1,9 +1,15 @@
+require "json"
```

Comment:
Is this actually needed?  I think we require it somewhere else but don't quote me.

Reply:
It was necessary due to the way I was testing. I've removed it.